gh-120254: Add a `commands` argument to `pdb.set_trace` (#120255)
python/cpython: Doc/library/pdb.rst

   The *readrc* argument defaults to true and controls whether Pdb will load
   .pdbrc files from the filesystem.

>    The *commands* argument, if given, would be a list of commands to execute
>    when the debugger starts. It has similar effects to the :file:`.pdbrc` file.

Reviewer: ```suggestion
   The *commands* argument, if given, is a list of commands to execute
   when the debugger starts. Those commands are executed after those
   optionally provided by the :file:`.pdbrc` file. 
```

Author: Do you think two "those" is a little bit confusing here? How about These commands are executed after any commands optionally provided by the :file:`.pdbrc` file?

Reviewer 2: I agree the double "those" is confusing.

Reviewer: Oups, you're right! I'm sorry I didn't reply to your question 3 weeks ago btw, but yes your suggestion is way better!

>    when the debugger starts. It has similar effects to the :file:`.pdbrc` file.

Reviewer 3: The phrase “similar effects” is kind of vague. I am not sure how to say it more precisely. Link to the `.pdbrc` documentation? Not sure.
